optimum/habana/transformers/models/gptj/modeling_gptj.py

```diff
@@ -77,10 +175,13 @@ def forward(
         Copied from GPTJAttention.forward: https://github.com/huggingface/transformers/blob/main/src/transformers/models/gptj/modeling_gptj.py
```

Comment:
@atakaha there is mismatch from the information. Here, you mentioned that you copied this piece of code from transformers master branch. At line 267, it is copied from 4.38.2. Please be consistent!

Reply:
This is from original and updated.